src/common/recordbatch/src/lib.rs

```diff
@@ -196,12 +196,17 @@ impl Stream for SimpleRecordBatchStream {
 pub struct RecordBatchStreamAdaptor {
     pub schema: SchemaRef,
     pub stream: Pin<Box<dyn Stream<Item = Result<RecordBatch>> + Send>>,
+    pub output_ordering: Option<Vec<OrderOption>>,
```

Comment:
Why do we need to wrap an `Option` on top of `Vec`? And what does `Some(Empty vec)` mean？

Reply:
This comes from https://docs.rs/datafusion/latest/datafusion/physical_plan/trait.ExecutionPlan.html#tymethod.output_ordering. `Some(Empty Vec)` should be the same with `None`